packages/client/src/pages/Account/components/PasswordEditor/PasswordEditor.module.scss

```
@@ -0,0 +1,7 @@
+@import '@styles/variables';
+
+.dialog {
+  background: $darkPurple;
```

Присоединясь к комменту о дизайн токенах:
```suggestion
  background: $backgroundColorDialog;
```